### PR TITLE
Translation corrections

### PR DESCRIPTION
**Pull request by Tymaeus**
Various translation corrections done here and there, mostly in the scenes involving Chisato and Tamako.

### CODE REVIEW

File: Update/some_002.txt [hunk 1]
```diff
@@ -3248,7 +3319,7 @@ void main()
 //rやられた、と思ったときにはもう遅い。二人はおなかを抱えて、げらげらと盛大に笑い声を上げていた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "やられた、と思ったときにはもう遅い。二人はおなかを抱えて、げらげらと盛大に笑い声を上げていた。",
-			NULL, "It's done, it's already too late. The two of them are holding their sides, laughing and giggling loudly.", Line_Normal);
+			NULL, "By the time I realize I've been had, it's already too late. The two of them are holding their sides, laughing and giggling loudly.", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/some_002.txt:3322]:
```suggestion
			NULL, "By the time I realize I'd been had, it was already too late. The two of them were holding their sides, laughing and giggling loudly.", Line_Normal);
```

File: Update/some_002.txt [hunk 2]
```diff
@@ -930,7 +1001,7 @@ void main()
 //rそれを見て唖然と固まる私と違って、割って入ってきた女の子は動じた素振りもなく辞書をぽんぽんと叩きながら、呆れたようにため息をついて言った。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "それを見て唖然と固まる私と違って、割って入ってきた女の子は動じた素振りもなく辞書をぽんぽんと叩きながら、呆れたようにため息をついて言った。",
-			NULL, "Unlike me, who was stunned just watching, the other girl broke in swinging a dictionary. Noticing my amazement, she sighed and then began to speak.", Line_Normal);
+			NULL, "Unlike me, who was stunned just watching, the other girl broke in swinging a dictionary. She sighed with exasperatio and then began to speak.", Line_Normal);
```
Comment (Tymaeus) [inline Update/some_002.txt:1004]: Sorry, my bad, typo. Should be "exasperation".

File: Update/some_002.txt [hunk 3]
```diff
@@ -3631,7 +3702,7 @@ void main()
 //r千紗登ちゃんのありがたい提案に、私は喜んで頷いた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "千紗登ちゃんのありがたい提案に、私は喜んで頷いた。",
-			NULL, "I nodded freely to Chisato-chan's gracious proposal.", Line_Normal);
+			NULL, "I nodded gladly to Chisato-chan's gracious proposal.", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/some_002.txt:3705]:
```suggestion
			NULL, "I cheerfully nodded to Chisato-chan's gracious proposal.", Line_Normal);
```

File: Update/some_002.txt [hunk 4]
```diff
@@ -872,7 +943,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300040", 256, TRUE);
 	OutputLine(NULL, "「——あ、なんならこの私がこの手で事前に、じっくりねっとり予備診断をぉほぉぉっっ——」",
-			NULL, "\"...Oh, why would I give a negative diagnosis this far in advaaaaaance...?\"", Line_Normal);
+			NULL, "\"...Oh, if you'd like, I could take my time giving you a preliminary diagnosis with my very handsssssss...?\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/some_002.txt:946]:
```suggestion
			NULL, "\"...Oh, if you'd like, I could take my time giving you a preliminary diagnosis with my very haaaaaands...?\"", Line_Normal);
```

**Pull request merged**